### ISSUE TITLE
Suggestion for new user creation

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#877]
I suggest that upon creation of a new user account the user should be directed to the manage profile page rather than the search page to encourage new users to enter their name and customize their account before starting to search.

**Comment by dustymc** [comment]
There are three issues addressing the same things open - I'm closing this as a duplicate of #878
